src/types/module.rs

```diff
@@ -133,13 +150,13 @@ impl PyModule {
     /// let code = std::fs::read_to_string("assets/script.py")?;
     ///
     /// Python::with_gil(|py| -> PyResult<()> {
-    ///     PyModule::from_code_bound(py, &code, "example.py", "example")?;
+    ///     PyModule::from_code(py, &code, "example.py", "example")?;
     ///     Ok(())
     /// })?;
     /// Ok(())
     /// # }
     /// ```
-    pub fn from_code_bound<'py>(
+    pub fn from_code<'py>(
```

Comment:
Same opportunity here

Reply:
This one it seems to me like changing to take `&CStr` is the right choice 👍

Reply:
`from_code` now accepts &CStr